ant/logback.xml

```diff
@@ -5,7 +5,7 @@
       <pattern>%d [%t] %-5p %c - %m%n</pattern>
     </encoder>
   </appender>
-  <root level="WARN">
+  <root level="DEBUG">
```

Comment:
Could you explain more about this change?

Reply:
An unintentional addition from a dirty tree; I've got rid of it.